node/node.go

```diff
@@ -267,18 +267,26 @@ func New(host p2p.Host, consensusObj *consensus.Consensus, db ethdb.Database, is
 
 		node.Consensus.VerifiedNewBlock = make(chan *types.Block)
 
+		// Add Faucet contract to all shards
+		if isFirstTime {
+			// Setup one time smart contracts
+			node.AddFaucetContractToPendingTransactions()
+		} else {
+			node.AddContractKeyAndAddress(scFaucet)
```

Comment:
write a comment here and below to clarify what we need to add for all shards here and for beacon chain below.